codalab/lib/codalab_manager.py

```diff
@@ -232,6 +243,34 @@ def init_config(self, dry_run=False):
 
         return config
 
+    def init_logger(self):
+        """
+        Initialize and configure the root logger.
+        Every module should get its own logger instance:
+            import logging
+            log = logging.getLogger(__name__)
+        """
+        logger = logging.getLogger()
+        logger.setLevel(logging.DEBUG)
+
+        # create file handler which logs even debug messages
+        log_file = self.config.get('logging', {}).get('file_path', os.path.join(self.codalab_home, 'codalab.log'))
```

Comment:
Being religious about the 80 character policy can sometimes make code even harder to read, I don't think we need to adhere to that.

That being said, one could definitely split this up by defining a default_file_path variable if one wanted.